File: 1-Iniciante/Ex1098.java
public class Ex1098 {
    public static void main(String[] args) {
        // processamento & saida
        float i = 0f, j = 1f;
        int cont = 0;

        while(i < 2.1){
            print(i, j);

            j += 1;
            cont++;
            if(cont == 3){
                i += 0.2d;
                j -= 2.8d;
                cont = 0;
            }
        }
    }

    public static void print(float i, float j){
        int partInt = (int)i;
        float aux = i - partInt;

        if(aux >= 0.2) System.out.printf("I=%.1f J=%.1f\n", i, j);
        else System.out.printf("I=%.0f J=%.0f\n", i, j);
    }
}

File: 1-Iniciante/ex1098.js
// saida
let j = 1.0;
for(let i = 0.0; i < 2.0; i += 0.2){
    for(let cont = 0; cont < 3; cont++, j++){
        const verificar_inteiro = (n) => {
            let num_str = n.toFixed(1) + '';
            if(num_str.charAt(2) === '0') return true;
            return false;
        };
        if(verificar_inteiro(i)) console.log(`I=${i.toFixed(0)} J=${j.toFixed(0)}`);
        else console.log(`I=${i.toFixed(1)} J=${j.toFixed(1)}`);
    }
    j -= 2.8;
}